### DISCUSSION TITLE
Amiberry on Windows x64 and on Windows on ARM and on Mac OS on ARM

### DISCUSSION THREAD

**Discussion by MaXGitteR** [BlitterStudio/amiberry#754] [Q&A] [answered]
Hello,

why isn't there a Amiberry Version on Windows 10 x64?
Yes I read this:  "Amiberry is an optimized Amiga emulator for ARM-based SoCs ..."
But would it be a big problem to compile/create/maintaine a Win x64 Version?
Or is it simply not wanted or not the focus of this project?

Will there be a Amiberry Version on Windows 10 on ARM and or Mac OS on ARM (Apple Silicon)?
You wrote: "Amiberry is an optimized Amiga emulator for ARM-based SoCs ...".
Ok not only for Linux, only the CPU Architecture is mentioned.
So there is a Windows 10 ARM and a MacOS Version which runs on ARM SoCs. Sounds like the focus of this project?

Regards
MaXGitteR

- The questions are intentionally asked in a special way,  but unfortunately I have not found anything on the website, the FAQ 
 and the open and closed issues on this. -

**Comment by solskogen** [comment, marked as answer]
Amiberry is only for ARM devices running Linux at the moment. Take a look at the documentation for supported devices.
The reason is that there haven't been any patches or pull requests for other architectures or operating systems. In other words, lack of developers. Patches are more than welcome!

**Comment by midwan** [comment]
As @solskogen said, the main issue is the requirement on someone maintaining that port.
Since x86/x64 platforms already had options (WinUAE on Windows, FS-UAE on the rest), the main focus of Amiberry has always been ARM-based devices. It is possible to bring back the WinUAE-based x86 code, so it can be eventually compiled on non-ARM platforms and run there, but it requires some work (mostly on the CPU-related parts and the ARM-optimized ones).

Regarding ARM based Windows and MacOS platforms, the same thing applies: developers to maintain it, as well as available platforms to test/develop with.

Since Amiberry so far is mostly a one man project, the above may happen at some time in the future anyway, but it will probably take quite a while before that happens. :)

**Comment by MaXGitteR** [comment]
Amiberry v8.0.0
Windows Native Support
Full native Windows version (https://github.com/BlitterStudio/amiberry/pull/1781)

**Reply by midwan** [reply]
Yeah, a few years later, and here we are :)
